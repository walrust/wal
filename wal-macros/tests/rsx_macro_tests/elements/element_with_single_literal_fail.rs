use wal_macros::rsx;

fn main() {
    rsx! { <div> b"val" </div> };
}
