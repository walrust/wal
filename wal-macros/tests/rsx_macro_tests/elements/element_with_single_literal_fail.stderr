error: Byte string literals are not supported
 --> tests/rsx_macro_tests/elements/element_with_single_literal_fail.rs:4:18
  |
4 |     rsx! { <div> b"val" </div> };
  |                  ^^^^^^
